Sources/Sentry/SentryProfiler.mm

```diff
@@ -522,6 +522,9 @@ - (void)captureEnvelope
     const auto profileDuration = getDurationNs(_startTimestamp, _endTimestamp);
     profile[@"duration_ns"] = [@(profileDuration) stringValue];
     profile[@"truncation_reason"] = profilerTruncationReasonName(_truncationReason);
+    profile[@"platform"] = _transactions.firstObject.platform;
+    profile[@"environment"] = _hub.scope.environmentString ?: _hub.getClient.options.environment ?: kSentryDefaultEnvironment;
+    profile[@"timestamp"] = [[SentryCurrentDate date] sentry_toIso8601String];
```

Comment:
Please confirm that a string such as `2022-11-02T21:16:20.877Z` would validate? @phacops

Reply:
It would.